__tests__/app.test.js

```diff
@@ -38,3 +39,41 @@ describe.only("3. GET /api/topics", () => {
       });
   });
 });
+
+describe("4. GET /api/articles", () => {
+  it("status:200, should responds with an array of articles with the creation date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeInstanceOf(Array);
+        expect(articles).toHaveLength(12);
+        expect(articles).toBeSorted("created_at", { descending: true });
+
+        articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              comment_count: expect.any(Number),
+            })
+          );
+        });
+      });
+  });
+
+  it("status:404, should responds with error message when the path is invalid", () => {
```

Comment:
This 404 test can go into a separate describe block (unless you have it already) as it is testing a general endpoint for any non-existent path.

Reply:
Oh okay but I tried to do it on every describe block so that it shows passed on all the invalid path